From: Antonio Radici <antonio@debian.org>
Date: Sun, 14 Aug 2016 22:07:44 +0100
Subject: disable_failing_test_on_sparc

===================================================================
---
 tests/unit/Makefile.am | 4 ----
 1 file changed, 4 deletions(-)

--- a/tests/unit/Makefile.am
+++ b/tests/unit/Makefile.am
@@ -156,7 +156,6 @@
 	rb-tree-test \
 	variable_test \
 	protocol_test \
-	mon_cpu_test \
 	mon_load_test \
 	mon_processes_test \
 	mustache_test \
@@ -391,9 +390,6 @@
 	../../libenv/libenv.la \
 	../../libpromises/libpromises.la
 
-mon_cpu_test_SOURCES = mon_cpu_test.c ../../cf-monitord/mon.h ../../cf-monitord/mon_cpu.c
-mon_cpu_test_LDADD = ../../libpromises/libpromises.la libtest.la
-
 mon_load_test_SOURCES = mon_load_test.c ../../cf-monitord/mon.h ../../cf-monitord/mon_load.c
 mon_load_test_LDADD = ../../libpromises/libpromises.la libtest.la
 
